<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1191</width>
    <height>496</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ASN.1 I/O</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="1" column="0" colspan="5">
       <widget class="QTextEdit" name="edtASN1"/>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="btnComp">
        <property name="text">
         <string>Compile</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnReset">
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ASN.1 Struct Input</string>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3" colspan="2">
       <widget class="QPushButton" name="btnOpenASN">
        <property name="text">
         <string>Open ASN1 Library</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="rbtEncode">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Encode</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="rbtDecode">
        <property name="text">
         <string>Decode</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="cbBox">
        <item>
         <property name="text">
          <string>UPER</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PER</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BER</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DER</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CPER</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UCPER</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CER</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>OER</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>COER</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QTextEdit" name="edtCode"/>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="btnCode">
        <property name="text">
         <string>Encode</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QTextEdit" name="edtOpt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Results</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QTextEdit" name="edtRes">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Console  Output</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>edtASN1</tabstop>
  <tabstop>btnReset</tabstop>
  <tabstop>btnComp</tabstop>
  <tabstop>rbtEncode</tabstop>
  <tabstop>rbtDecode</tabstop>
  <tabstop>cbBox</tabstop>
  <tabstop>edtCode</tabstop>
  <tabstop>btnCode</tabstop>
  <tabstop>edtOpt</tabstop>
  <tabstop>edtRes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
